packages/story-editor/src/app/media/utils/getResourceFromAttachment.js

```diff
@@ -52,7 +52,6 @@ function getImageResourceFromAttachment(attachment) {
     id,
     date_gmt,
     media_details: { width, height, sizes },
-    title: { raw: title },
```

Comment:
Since we are no longer using this field, it means we can stop fetching that from the REST API.

That means removing these two lines:

https://github.com/google/web-stories-wp/blob/c6432948c1a91f42ca19b1069255281fff43dd0a/packages/story-editor/src/app/api/apiProvider.js#L239

https://github.com/google/web-stories-wp/blob/c6432948c1a91f42ca19b1069255281fff43dd0a/includes/templates/admin/edit-story.php#L88

Reply:
Can't we just leave those fixutres as-is _and_ remove the two lines I mentioned? I don't see why that wouldn't work.

Description is the best example: we don't request the field but it's in the mock response. That's totally fine.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/8613/commits/e9efdfb70fa2a0bd0a9851cb4c224a4c144393ea